packages/paste-core/components/form-pill-group/__tests__/index.spec.tsx

```diff
@@ -44,6 +45,21 @@ const I18nProp = (): JSX.Element => {
   );
 };
 
+describe("calculateSizeStyles", () => {
+  it("should calculate the default style when passed default. columnGap: space20, rowGap: space20", () => {
+    const { columnGap, rowGap } = calculateSizeStyles("default");
+    expect(columnGap).toEqual("space20");
+    expect(rowGap).toEqual("space20");
+  })
+
+  it("should calculate the large sytyle when passed L. columnGap: space30, rowGap: space30, fontSize: fontSize30", () => {
+    const { columnGap, rowGap, fontSize } = calculateSizeStyles("L");
+    expect(columnGap).toEqual("space30");
+    expect(rowGap).toEqual("space30");
+    expect(fontSize).toEqual("fontSize30");
+  })
+})
```

Comment:
I tried testing this more directly using spies, but encountered some jest issues based on the fact that the module was treating the "Box" component as a getter function. For some reason our Jest can't determine that it is in fact a function and not an object. Exporting the function allows us to still test it, but in a bit of a less ideal way.